doc/api/tls.md

```diff
@@ -1578,6 +1578,14 @@ secureSocket = tls.TLSSocket(socket, options);
 
 where `secureSocket` has the same API as `pair.cleartext`.
 
+### tls.roots
```

Comment:
Currently, this section is placed in the deprecated API section. It needs to be placed after `## tls.getCiphers()` with level 2 (`##`) heading.

`## tls.DEFAULT_ECDH_CURVE` also seems to be out of ABC order, but this can be addressed in a different PR.

Reply:
Maybe `tls.DEFAULT_CA` would align better with the other `tls.DEFAULT_...` values? Docing DEFAULT_MAX/MIN_VERSION is almost at the  top of my TODO list, and using similar name patterns for the defaults would cluster these together in the docs, and emphasize the connection between the `ca: ` option and the `tls.XXX` value.